### PR TITLE
delegate metric updates

### PR DESCRIPTION
**Pull request by mkrorbrkr**
updated metrics for delegates as of 25/11/21 @LongForWisdom

### CODE REVIEW

File: governance/delegates/0xB0B829a6AaE0F7e59B43391b2C8a1CFD0C801c8C/metrics.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-combined_participation: No Data
-poll_participation: No Data
+combined_participation: 0.00%
+poll_participation: 0.00%
 exec_participation: No Data
 communication: No Data
 start_date: 2030-01-01T00:00:00.000Z
```
Comment (LongForWisdom) [inline governance/delegates/0xB0B829a6AaE0F7e59B43391b2C8a1CFD0C801c8C/metrics.md:6]: Need to update gauntlet's start date!

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

File: governance/delegates/0xf60D7a62C98F65480725255e831DE531EFe3fe14/metrics.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-combined_participation: No Data
-poll_participation: No Data
+combined_participation: 100%
+poll_participation: 100%
 exec_participation: No Data
-communication: No Data
+communication: 100%
 start_date: 2030-01-01T00:00:00.000Z
```
Comment (LongForWisdom) [inline governance/delegates/0xf60D7a62C98F65480725255e831DE531EFe3fe14/metrics.md:6]: Update start date!

File: governance/delegates/0xB0B829a6AaE0F7e59B43391b2C8a1CFD0C801c8C/metrics.md (hunk 1 above)
Comment (mkrorbrkr) [reply governance/delegates/0xB0B829a6AaE0F7e59B43391b2C8a1CFD0C801c8C/metrics.md]: fixed

File: governance/delegates/0xf60D7a62C98F65480725255e831DE531EFe3fe14/metrics.md (hunk 2 above)
Comment (mkrorbrkr) [reply governance/delegates/0xf60D7a62C98F65480725255e831DE531EFe3fe14/metrics.md]: fixed

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**